### PR TITLE
Coteries

### PR DESCRIPTION
**Pull request by ubermichael**
Add coteries.

### CODE REVIEW

File: templates/image_file/index.html.twig [hunk 1]
```diff
@@ -4,13 +4,13 @@
```
Comment (joeytakeda) [inline templates/image_file/index.html.twig:4]: Making a note here that we should probably remove all of the image_file stuff at some point; they're not used and I don't believe there are any plans to do so.

File: templates/manuscript/partial/gallery-item.html.twig [hunk 2]
```diff
@@ -14,38 +14,40 @@ either in the body of a page or within the contents of a tab #}
...
-                    <a href="{{ path('period_show', { 'id': period.id}) }}">
-                        {{ period }}
-                    </a>
-                    {%- if not loop.last %}, {% endif %}
-                    {% endfor %}
-                </td>
-            </tr>
+                <tr>
+                    <th>Period</th>
+                    <td>{% for period in manuscript.periods|sort((a,b) => a.label <=> b.label) %}
```
Comment (joeytakeda) [inline templates/manuscript/partial/gallery-item.html.twig:19]: Nice catch :-)

File: templates/manuscript/show.html.twig [hunk 3]
```diff
@@ -18,268 +18,274 @@
...
+                    <tr>
+                        <th>First Line</th>
+                        <th>Context</th>
+                        {% if showPrintSource %}
+                            <th>Print Source</th>
+                        {% endif %}
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for manuscriptContent in manuscript.manuscriptContents|sort((a,b) => a.content.firstLine <=> b.content.firstLine) %}
```
Comment (joeytakeda) [inline templates/manuscript/show.html.twig:228]: Not requesting a change here, but we should probably check with MVM to see if they'd prefer sorted by firstLine or by date (though I suppose firstLine is more reliable)

**Comment by ubermichael** [comment]
They mentioned first line a few times.

**Review by joeytakeda — COMMENTED** [review]
This looks good to me and I think it ought to be merged. The summary below is basically just a list of items for discussion next time we meet with the project team; I'll make these into tickets once this is merged.

Summary:

* Do we need the `image_*` stuff? It's not used currently, so it might be nice to get clarification as to whether that will ever happen
* We should really be using the standard gallery for all manuscripts; right now, we use gallery and gallery item partials in order to provide empty divs in order to align all boxes properly in a grid view. We should really be using grid for that, which solves the problem much more elegantly and would mean that we could define the manuscript sorting in a single place (if that's desired)
* Once this is merged, the various coterie pages will need to be refashioned to align with the rest of the site (i.e. breadcrumbs, etc).
* We'll need clarification as to where coteries should go / how they should be displayed (i.e. just like people, or should it operate differently?)

**Pull request merged**